docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -109,9 +114,20 @@ Events are categorized based on the `event.category` value.
 [role="screenshot"]
 image::images/event-type.png[]
 
-When you select an `event.category` pill, all the events within that category are listed in the left panel. To view more details about a specific event, select it from the list.
+When you select an `event.category` pill, all the events within that category are listed in the left panel. To display more details about a specific event, select it from the list.
 
 [role="screenshot"]
 image::images/event-details.png[]
 
 NOTE: In {stack} versions 7.10.0 and newer, there is no limit to the number of events that can be associated with a process. However, in {stack} versions 7.9.0 and earlier, each process is limited to only 100 events.
+
+To examine alerts associated with the event, select the alert pill (*_x_ alert*). The left pane lists the total number of associated alerts, and alerts are ordered from oldest to newest. Each alert shows the type of event that produced it (`event.category`), the event timestamp (`@timestamp`), and rule that generated the alert (`kibana.alert.rule.name`). Click on the rule name to open the alert's details.
+
+In the example screenshot below, five alerts were generated against the analyzed event (`lsass.exe`). The left pane displays the associated alerts and basic information about each one.
+
+preview::[]
+
+NOTE: This is a https://www.elastic.co/pricing[Platinum or Enterprise subscription] feature. In addition, to display it in {elastic-security} you must add the `xpack.securitySolution.enableExperimental: ['insightsRelatedAlertsByProcessAncestry']` feature flag to the `kibana.yml` file.
```

Comment:
Attribute correx. Also I think it's more readable to include long code references at the end of a sentence, instead of breaking up the flow in the middle:
```suggestion
NOTE: This is a https://www.elastic.co/pricing[Platinum or Enterprise subscription] feature. In addition, to display it in {elastic-sec} you must add the following feature flag to the `kibana.yml` file: `xpack.securitySolution.enableExperimental: ['insightsRelatedAlertsByProcessAncestry']`
```